x-pack/plugins/ml/server/models/data_recognizer/modules/security_auth/ml/auth_rare_user.json

```diff
@@ -1,23 +1,17 @@
 {
   "description": "Security: Authentication - looks for an unusual user name in the authentication logs. An unusual user name is one way of detecting credentialed access by means of a new or dormant user account. A user account that is normally inactive, because the user has left the organization, which becomes active, may be due to credentialed access using a compromised account password. Threat actors will sometimes also create new users as a means of persisting in a compromised web application.",
```

Comment:
Probably worth changing the case of `looks` in this set of edits for consistency to `Looks`.

Reply:
Changed!